lib/views/frontend/spree/checkout/payment/_paypal_commerce_platform.html.erb

```diff
@@ -1,3 +1,8 @@
+<script>
+  Spree.current_order_id = "<%= @order.number %>"
```

Comment:
Can we move the order id and token setters into the [existing script tag](https://github.com/solidusio-contrib/solidus_paypal_commerce_platform/blob/610c00ec8414408ca73f1225ef1bc18d71eddc56/lib/views/frontend/spree/checkout/payment/_paypal_commerce_platform.html.erb#L7)? Just to keep with the precedent set by the `orders/payment` file.

Reply:
Definitely. On it!